index.html

```diff
@@ -380,20 +380,24 @@ <h2 class="opinion_me">I'm always interested in hearing about new
             <ul>
                 <li>
                     <input class="form_input" name="fname" type="text" placeholder="Full Name" maxlength="30" required>
+                    
                 </li>
                 <li>
-                    <input class="form_input" name="email" type="email" placeholder="Email address" required>
+                    <input id="mail" class="form_input email_input" name="email" type="email" placeholder="Email address" required>
                 </li>
-                <li>
-                    <textarea class="form_input textarea" name="textarea" placeholder="Enter text here" maxlength="500"
+                <li class="error"> <span> The email must be in lowercase letters </span></li>
+                <li >
```

Comment:
- [ ] Please ensure that the error message displays close to the button and also ensure that it does not disappear until the user corrects the email